multicluster/controllers/multicluster/serviceexport_controller.go

```diff
@@ -313,15 +348,18 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 	now := metav1.Now()
 	var res, message *string
 	switch cause {
-	case notFound:
+	case serviceNotFound:
 		res = getStringPointer("not_found_service")
 		message = getStringPointer("the Service does not exist")
-	case importedService:
+	case serviceWithoutEndpoints:
+		res = getStringPointer("no_endpoints_service")
```

Comment:
"no_endpoints" or "service_without_endpoints"

Reply:
Changed to "service_without_endpoints"